handlers/middlewares/logPresence.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+const config = require('../../config');
+function getUsername(user) {
+	var str = "";
+	if (user.first_name) str += user.first_name;
```

Comment:
No need for the `if`, `first_name` is not optional.
```suggestion
	let str = user.first_name;
```